packages/logconsole-extension/schema/plugin.json

```
@@ -52,8 +64,70 @@
         { "const": "debug", "title": "Debug" }
       ],
       "default": "warning"
+    },
+    "toolbar": {
+      "title": "Log console toolbar items",
+      "description": "Note: To disable a toolbar item,\ncopy it to User Preferences and add the\n\"disabled\" key. The following example will disable the uploader button:\n{\n  \"toolbar\": [\n    {\n      \"name\": \"uploader\",\n      \"disabled\": true\n    }\n  ]\n}\n\nToolbar description:",
```

Also updating the example to make sense in this context:

```suggestion
      "description": "Note: To disable a toolbar item,\ncopy it to User Preferences and add the\n\"disabled\" key. The following example will disable the clear button:\n{\n  \"toolbar\": [\n    {\n      \"name\": \"clear\",\n      \"disabled\": true\n    }\n  ]\n}\n\nToolbar description:",
```